SU-HSDO/stanford_mrc
# MRCD8-175 visitor node

# READY FOR REVIEW

# Summary
- Visitor node styles
- Updated Gruntfile.js for stanford_basic alongside MRC theme and recompiled CSS to fix theme issue

# Needed By (Date)
- End of sprint (2/1)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Test the node display
3. Verify that Decanter grid classes are now working

# Affected Projects or Products
- MRC
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-175

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
